tools/serve/serve.py

```diff
@@ -940,6 +940,7 @@ def get_parser():
     parser.add_argument("--no-h2", action="store_false", dest="h2", default=None,
                         help="Disable the HTTP/2.0 server")
     parser.add_argument("--quic-transport", action="store_true", help="Enable QUIC server for WebTransport")
+    parser.add_argument("--exit-after-start", action="store_true", help="Exit after starting servers")
```

Comment:
I could definitely see switching this to something like `--build-config-only` and early-exiting the loop on line 955. Currently it does start the servers, but it doesn't really check anything so may be rather useless (for any other purpose than regenerating the certs).

Reply:
Hrm, that may actually be better as well, given the failures in my testing run: https://github.com/stephenmcgruer/wpt/runs/1047841369?check_suite_focus=true

Not sure why the stash server is failing there.

Reply:
Did you set up /etc/hosts? I'm guessing that might be why (Stash trying to bind to e.g. web-platform.tests and get DNS failures.)

Reply:
Whoops, that's exactly it. Adding setup seems to fix it (see e.g. https://github.com/stephenmcgruer/wpt/actions/runs/234768815)